packages/shared/src/lib/featureManagement.ts

```diff
@@ -128,10 +128,17 @@ export class Features {
     ['control', 'v1', 'v2', 'v3'],
   );
 
-  static readonly MyFeedVersion = new Features('my_feed_version', 'v1', [
+  static readonly FeedFilterVersion = new Features(
```

Comment:
Be aware the old version should still be active and is a separate AB test to see if that one is shown or the new one.

Reply:
Yes, this change is relevant from the intermediate branch, the original should have a different variable. Just thought of renaming the new one to avoid confusion since the original one is named `MyFeedOnboardingVersion` and both are for onboarding.